crates/gpio/src/gpio.rs

```diff
@@ -413,11 +413,11 @@ impl DummyDevice {
 }
 
 impl GpioDevice for DummyDevice {
-    fn open(_device: u32) -> Result<Self>
+    fn open(ngpios: u32) -> Result<Self>
     where
         Self: Sized,
     {
-        Ok(DummyDevice::new(8))
+        Ok(DummyDevice::new(ngpios.try_into().unwrap()))
```

Comment:
Maybe DummyDevice::new() should accept u32 now instead of u16 ?

Reply:
that does cause a ripple of changes to other interfaces though.